hasura/migrations/default/1698332034738_alter_table_public_poap_events_add_column_distance/up.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."poap_events" add column "distance" float8 null;
```

Comment:
So this was added as a "synthetic" column just to be used for the vector search functions to put the similarity value somewhere. The way hasura works is it needs the column to exist in the schema so I had to add this. But it should be renamed similarity not distance now since we're using cosine similarity not distance. @CryptoGraffe